css/selectors.json

```diff
@@ -342,14 +342,14 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:link"
   },
-  ":matches": {
-    "syntax": ":matches( <selector># )",
+  ":is": {
+    "syntax": ":is( <complex-selector># )",
```

Comment:
I’m sort to bug you a third time, but I realised that this should actually be `<complex-selector-list>`, like in https://github.com/mdn/data/pull/287: https://github.com/mdn/data/blob/83365d0f7c8665c2fb143d157bf0dc47ac598cec/css/selectors.json#L354